src/test/tests/databases/export_db.py

```diff
@@ -114,7 +113,10 @@ def test0():
 #    CloseDatabase(silo_data_path("wave.visit"))
 
 def VTK_check_binary(filename):
-    f = open(filename, "rt")
+    if (sys.version_info > (3, 0)):
```

Comment:
On python 3 are we really *obligated* to pass these additional args? Won't `open(filename, "rt")` work correctly in both, perhaps with slightly different error behavior?

Reply:
it does fail so we need the extra options:
```
Traceback (most recent call last):
  File "/Users/harrison37/Work/github/visit-dav/visit/src/test/tests/databases/export_db.py", line 557, in <module>
    main()
  File "/Users/harrison37/Work/github/visit-dav/visit/src/test/tests/databases/export_db.py", line 545, in main
    test1()
  File "/Users/harrison37/Work/github/visit-dav/visit/src/test/tests/databases/export_db.py", line 183, in test1
    if VTK_check_binary("binary_VTK/binary_VTK.0.vtk"):
  File "/Users/harrison37/Work/github/visit-dav/visit/src/test/tests/databases/export_db.py", line 123, in VTK_check_binary
    line = f.readline()
  File "/Users/harrison37/Work/github/visit-dav/visit/build-mb-develop-darwin-10.14-x86_64/build-debug-py3/lib/python/lib/python3.7/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd5 in position 142: invalid continuation byte
```

The reason why is all text files are read using an encoding. This VTK file is actually binary, we are reading the first line. But if we don't tell python to ignore encoding errors, it fails.

Reply:
Ok, fair enough and too bad.